lib/__tests__/fixtures/cache/valid.css

```diff
@@ -0,0 +1 @@
+/* This will not cause a CSS syntax error */
```

Comment:
Do you mean this will not cause a *linting* error? Those "invalid" files also will not cause CSS syntax errors, right?